### ISSUE TITLE
[FATAL] [transports/janus_http.c:janus_http_init:854] Couldn't start secure webserver on port 8089...

### ISSUE THREAD

**Issue by Mangomm** [meetecho/janus-gateway#3192]
my janus version:  v1.1.3

When I ran janus, the audio and video call was OK, but there was a title error, which I could not accept. I studied the janus source code, it may be MHD_start_daemon() problem.

my janus.transport.http.jcfg：
![image](https://user-images.githubusercontent.com/58781546/228508981-a4f90262-263f-4b7e-966a-1dc54633ee3a.png)
![image](https://user-images.githubusercontent.com/58781546/228509011-101a8774-cd82-49b3-951d-3963a1c7be78.png)

janus debug info:
![image](https://user-images.githubusercontent.com/58781546/228510671-5c713fdf-b799-4fba-b226-449a338046f2.png)

**Comment by lminiero** [comment]
Your MHD library was probably compiled without encryption support.

**Comment by lminiero** [comment]
You can set `mhd_debug = true` in the transport config to see if it prints more precise info.

**Comment by Mangomm** [comment]
yes, i run  /opt/janus/bin/janus --debug-level=5 --log-file=$HOME/janus-log

**Comment by lminiero** [comment]
That's not the debug level I'm talking about. The MHD transport plugin has a different logging configuration.

**Comment by Mangomm** [comment]
Hi, I had some time to look into this error today and I am sure libmicrohttpd is https supported and can be seen by running ./configure. And then I got libmicrohttp example verification (libmicrohttpd 0.9.71 / SRC/examples / / benchmark_https. C), call MHD_start_daemon () is a success, But I don't know why janus failed.
![image](https://user-images.githubusercontent.com/58781546/229745742-78c29f7e-2ca9-4202-855a-639611bfe687.png)
![image](https://user-images.githubusercontent.com/58781546/229745936-cf35287a-73b6-41f8-8505-269b5d260a5d.png)


I thought it was the certificate problem, but I applied the successful certificate to janus, but it still failed.

One more thing, janus in "sudo vim janus. Transport. HTTP. JCFG" configuration, the mhd_debug = true doesn't work,
![image](https://user-images.githubusercontent.com/58781546/229746098-192e8100-106c-451f-923a-c449e2ea5f83.png)
